lib/sycamore/sycamore/tests/integration/test_html_to_opensearch.py

```diff
@@ -25,7 +26,7 @@ def test_html_to_opensearch():
             "settings": {"index.knn": True, "number_of_shards": 5, "number_of_replicas": 1},
             "mappings": {
                 "properties": {
-                    "embeddings": {
+                    "embedding": {
```

Comment:
I'm dumb. Why did this change? If it was wrong before how was it working?

Reply:
Before it was working by auto-creating a field called "embedding" that contains a list of floats. There are no vector queries in these tests so nothing broke. Now that we validate index configuration (targetparams compat check) this caused an error - when the ITs were not cleaning up after themselves and one of them had it configured correctly